wolfcrypt/src/asn.c

```diff
@@ -18020,33 +18080,42 @@ int ConfirmSignature(SignatureCtx* sigCtx,
                     word32 idx = 0;
             #if defined(WC_ECC_NONBLOCK) && defined(WOLFSSL_ASYNC_CRYPT_SW) && \
                 defined(WC_ASYNC_ENABLE_ECC)
+                #ifndef WOLFSSL_NO_MALLOC
                     ecc_nb_ctx_t* nbCtx;
+                #else
+                    ecc_nb_ctx_t nbCtx[1];
```

Comment:
ECC non-blocking will not be able to support a no malloc like this, since you cannot use a stack pointer and return later and have it available still. The customer is not using nonblocking ECC so please revert this code.

Reply:
Test case to reproduce: `./configure --enable-ecc=nonblock --enable-sp=yes,nonblock CFLAGS="-DWOLFSSL_PUBLIC_MP -DWOLFSSL_NO_MALLOC" && make && ./wolfcrypt/test/testwolfcrypt`.